README.md

```diff
@@ -4,8 +4,9 @@ Uniform interface for positive definite matrices of various structures.
 
 [![Build Status](https://travis-ci.org/JuliaStats/PDMats.jl.svg?branch=master)](https://travis-ci.org/JuliaStats/PDMats.jl)
 [![Coverage Status](https://img.shields.io/coveralls/JuliaStats/PDMats.jl.svg)](https://coveralls.io/r/JuliaStats/PDMats.jl?branch=master)
-[![PDMats](http://pkg.julialang.org/badges/PDMats_0.5.svg)](http://pkg.julialang.org/?pkg=PDMats&ver=0.5)
 [![PDMats](http://pkg.julialang.org/badges/PDMats_0.6.svg)](http://pkg.julialang.org/?pkg=PDMats&ver=0.6)
+[![PDMats](http://pkg.julialang.org/badges/PDMats_0.7.svg)](http://pkg.julialang.org/?pkg=PDMats&ver=0.7)
+[![PDMats](http://pkg.julialang.org/badges/PDMats_1.0.svg)](http://pkg.julialang.org/?pkg=PDMats&ver=1.0)
```

Comment:
Would the links stay the same for NewPkgEval? If so, I think it would not hurt much if 0.7 badge status would stay but be outdated; at least people will immediately see that the package is 0.7-aware. I will remove 1.0, because it looks ugly.

Reply:
I assume the links will be the same but I have no evidence either way. The only issue with keeping the outdated 0.7 badge is that if it shows that the package is failing on 0.7, it will look bad and be stuck like that. :stuck_out_tongue: I'll leave it up to you, do what feels right.